### PR TITLE
Experimental: add integration test

### PR DESCRIPTION
**Pull request by xlqian**

### CODE REVIEW

**Comment by TeXitoi** [comment]
Jenkins, test this please.

File: source/jormungandr/tests/routing_tests_experimental.py [hunk 1]
```diff
@@ -134,8 +134,30 @@ def test_crow_fly_sections(self):
...
+        query = "journeys?from={from_sa}&to={to_sa}&datetime={datetime}"\
+            .format(from_sa='stopA', to_sa='stop_point:stopB', datetime="20120614T080000")
+        response = self.query("v1/coverage/main_routing_test/" + query)
+        check_journeys(response)
+        jrnys = response['journeys']
+
+        j = next((j for j in jrnys if j['type'] == 'non_pt_walk'), None)
+        assert j
+        assert j['sections'][0]['from']['id'] == 'stopA'
+        assert j['sections'][0]['to']['id'] == 'stop_point:stopB'
```
Comment (TeXitoi) [inline source/jormungandr/tests/routing_tests_experimental.py:229]: You should check that the type is walking.

**Review by TeXitoi — CHANGES_REQUESTED** [review]

**Comment by kinnou02** [comment]
This PR has been waiting for two weeks!

**Comment by xlqian** [comment]
@kinnou02 thx for reminding ...

**Review by TeXitoi — APPROVED** [review]

**Pull request merged**